packages/pointer_interceptor/pointer_interceptor_ios/example/pubspec.yaml

```diff
@@ -0,0 +1,25 @@
+name: pointer_interceptor_ios_example
+description: "Demonstrates how to use the pointer_interceptor_ios plugin."
+publish_to: 'none'
+
+environment:
+  sdk: '>=3.1.0 <4.0.0'
+  flutter: '>=3.13.0'
+
+dependencies:
+  cupertino_icons: ^1.0.2
+  flutter:
+    sdk: flutter
+  pointer_interceptor:
```

Comment:
This should be removed.

Reply:
i dont think I can remove this - the integration tests fail because the swift tests require the pointer_interceptor package.

Reply:
What's the dependency on the Swift side that's specific to the app-facing package?

Reply:
I think its specific to the example? If i remove the this package from the pubspec it says it cant find the package on the swift side

Reply:
I think it just needs a dependency on pointer_interceptor_ios at least. Ill change it.

Reply:
Sorry, I missed that it didn't have one. The example should definitely depend on the platform it's an example for.